### DISCUSSION TITLE
Build request with commit 2219

### DISCUSSION THREAD

**Discussion by ghost** [Sporif/dxvk-async#22] [Q&A] [answered]
@Sporif 

Hello good sir.
I was wanting to test out this new commit from Joshie Async Patched. I think it may improve fps on CPU bound scenes. It's commit #2219 which can be found here 

https://github.com/doitsujin/dxvk/pull/2219

Would you kindly build an Async version including the changes in that commit.

Many thanks 🙏🙏

**Comment by Sporif** [comment, marked as answer]
Here you go: [dxvk-async-1.9.1_r31_ge84e1c0d.tar.gz](https://github.com/Sporif/dxvk-async/files/6951048/dxvk-async-1.9.1_r31_ge84e1c0d.tar.gz)

**Reply by ghost** [reply]
Thank you ☺️
